infrastructure/k8s-config/clusters/kit-infrastructure/perfdash/perfdash.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: perfdash
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: perfdash-config
+  namespace: perfdash
+  labels:
+    app.kubernetes.io/name: perfdash
+    app.kubernetes.io/component: config
+    app.kubernetes.io/part-of: perfdash
+data:
+  jobs.yaml: |
+    periodics:
+      - name: "kit-eks-1k"
+        tags:
+          - "perfDashPrefix: 1K test"
+          - "perfDashJobType: performance"
+      - name: "kit-eks-2k"
+        tags:
+        - "perfDashPrefix: 2k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-3k"
+        tags:
+          - "perfDashPrefix: 3K test"
+          - "perfDashJobType: performance"
+      - name: "kit-eks-36k"
+        tags:
+        - "perfDashPrefix: 36k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-4k"
+        tags:
+        - "perfDashPrefix: 4k test"
+        - "perfDashJobType: performance"
+      - name: "kit-eks-5k"
+        tags:
+          - "perfDashPrefix: 5K test"
+          - "perfDashJobType: performance"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: perfdash
+  namespace: perfdash
+  labels:
+    app.kubernetes.io/name: perfdash
+    app.kubernetes.io/version: "2.44"
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/part-of: perfdash
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: perfdash
+      app.kubernetes.io/version: "2.44"
+      app.kubernetes.io/component: dashboard
+      app.kubernetes.io/part-of: perfdash
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: perfdash
+        app.kubernetes.io/version: "2.44"
+        app.kubernetes.io/component: dashboard
+        app.kubernetes.io/part-of: perfdash
+    spec:
+      containers:
+        - name: perfdash
+          image: gcr.io/k8s-staging-perf-tests/perfdash:2.44
+          command:
+            - /perfdash
+            - --www=true
+            - --address=0.0.0.0:8080
+            - --builds=20
+            - --force-builds
+            - --mode=s3
+            - --aws-region=us-west-2
+            - --configPath=/etc/config/jobs.yaml
+            - --logsBucket=kit-eks-scalability
```

Comment:
Either we need to make sure we explicitly document this detail because other customers outside of us won't have access to this bucket and this deployment pods might fail or throw errors in the logs (OR) move this to internal repo ? Or patch this specific detail through flux  by taking in as param through cdk command line args ?

Reply:
I don't quite understand this comment.
Who is other customers? Other EKS teams or EKS customers?
IIUC this is bucket is only accessible to our team. Others can't access it no matter where we keep this yaml file.
Can you please elaborate your concern? We can discuss offline if you want.

Reply:
Updated according to our offline discussion.